src/browser/components/TeamListItem.jsx

```diff
@@ -9,9 +9,11 @@ class TeamListItem extends React.Component {
   }
 
   handleTeamRemove() {
+    document.activeElement.blur();
```

Comment:
In the team list, `TeamList.jsx` controls modals. So I think it should call document.activeElement.blur().